lib/errors.js

```diff
@@ -86,12 +86,15 @@ util.inherits(errors.ResourceNotFoundError, errors.SmartcarError);
  *
  * @extends SmartcarError
  * @param message - an error description to set
+ * @param body - A response body of Smartcar API call
+ * @param body.code - a vehicle state error code
  */
-errors.VehicleStateError = function(message) {
+errors.VehicleStateError = function(message, body) {
```

Comment:
Would it be better to have the code here since we're not using the body? No need to pass in more than we need right?

Reply:
Agree, will change it!